ci-operator/step-registry/hypershift/aws/e2e/nested/hypershift-aws-e2e-nested-workflow.yaml

```diff
@@ -31,4 +31,6 @@ workflow:
     pre:
     - ref: ipi-install-rbac
     - chain: hypershift-aws-setup-nested-management-cluster
+    - ref: hypershift-install-initial
```

Comment:
Is there a way to simply call the existing `hypershift-install` step with a different `HYPERSHIFT_RELEASE_LATEST` value? That would help to keep the code DRY

Reply:
it might be possible, but the problem is that we also need to override the `from` image so we also use the initial version of the CLI. This is not possible afaik.